Jit flush message fix

wingo opened the pull request.

wingo committed:
More helpful jit.flush message

wingo committed:
Fix time printout.

kbara commented:
I think it would be better to not have the tuning tip be conditional; if there are a lot of jit.flushes, it's easy to miss.

wingo committed:
Print troubleshooting link at every jit.flush.

kbara commented:
LGTM, though the lack of CI is bothersome.

wingo commented:
Fixed; now prints e.g.
```
336073.38695098: warning: Dropped more than 100000 packets; flushing JIT to try to recover.  See https://github.com/Igalia/snabb/blob/lwaftr_starfruit/src/program/lwaftr/doc/README.performance.md for performance tuning tips.
```
